seatunnel-server/seatunnel-app/pom.xml

```diff
@@ -479,7 +524,7 @@
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
             <version>${mysql.version}</version>
-            <scope>test</scope>
+            <scope>provided</scope>
```

Comment:
`mysql-connector-java` license is GPL, So, must use scope `test` here.

Reply:
thx，I  got it.

Reply:
> `mysql-connector-java` license is GPL, So, must use scope `test` here.

I have replaced "test" with "provided", But it does not work?

Reply:
I updated the pom.xml

Reply:
thx,  Merging is still blocked.